packages/vue-client/src/components/boards/MulticolorGraphBoard.vue

```diff
@@ -22,6 +23,32 @@ const intersections = computed(() =>
   createBoard(props.board_config, Intersection),
 );
 const graph = computed(() => createGraph(intersections.value, null));
+
+const voronoiDiagram = computed(() => {
+  const sites: Site[] = intersections.value.map((vector, index) => ({
+    id: index,
+    x: vector.position.X,
+    y: vector.position.Y,
+  }));
+
+  return new Voronoi().compute(sites, {
+    xl: viewBox.value.minX - 0.5,
+    xr: viewBox.value.minX + viewBox.value.width + 0.5,
+    yt: viewBox.value.minY - 0.5,
+    yb: viewBox.value.minY + viewBox.value.height + 0.5,
+  });
+});
+
+function getPolygonPointsString(cell: Cell): string {
+  return cell.halfedges
```

Comment:
The docs of voronoijs mention that it's possible that halfedges is empty array, which means that no cell could be calculated for this point. We may want to catch that case somehow, though for now I've decided to keep it simple.

Reply:
I'm curious what scenario halfedges would be empty.. it must be rare, right?

Reply:
That is my assumption. Maybe when two points have the exact same coordinates, or something like that.